lua/config/lazy.lua

```diff
@@ -18,7 +18,8 @@ if USE_STABLE then table.insert(spec, { import = "astronvim.lazy_snapshot" }) en
 require("lazy").setup {
   spec = vim.list_extend(spec, {
     -- AstroCommunity import any community modules here
-    -- { "AstroNvim/astrocommunity" },
+    -- TODO: Remove branch v4-refactor on release
+    { "AstroNvim/astrocommunity", branch = "v4-refactor" },
```

Comment:
```suggestion
    { "AstroNvim/astrocommunity", branch = "v4" },
```

This should just be v4 to align the branch names with all of the other repos